src/java_binding/java/java-binding/src/main/java/com/risingwave/java/binding/Binding.java

```diff
@@ -0,0 +1,27 @@
+package com.risingwave.java.binding;
+
+public class Binding {
+    static {
+        System.loadLibrary("risingwave_java_binding");
+    }
+
+    // iterator method
+    // Return a pointer to the iterator
+    static native long iteratorNew();
+
+    // return a pointer to the next row
+    static native long iteratorNext(long pointer);
+
+    static native void iteratorClose(long pointer);
```

Comment:
I understand the intention here but to make it clearer for others, can you add some comments to explain why we need a explicit `iteratorClose` rather than `drop`?